### ISSUE TITLE
Exception occurred when keyword argument default value is sys.stdout

### ISSUE THREAD

**Issue by sonofusion82** [pdoc3/pdoc#133]
Example:

    def test(a, b=sys.stdout):
        pass

will cause exception:

    File "C:\Users\xxx\AppData\Local\Programs\Python\Python37\lib\site-packages\pdoc\__init__.py", line 1138, in _params
        p = safe_default_value(p)
    File "C:\Users\xxx\AppData\Local\Programs\Python\Python37\lib\site-packages\pdoc\__init__.py", line 1111, in safe_default_value
        replacement = html.escape(replacement or p.default)
    File "C:\Users\xxx\AppData\Local\Programs\Python\Python37\lib\html\__init__.py", line 19, in escape
        s = s.replace("&", "&amp;") # Must be done first!
    AttributeError: '_io.TextIOWrapper' object has no attribute 'replace'

**Comment by kernc** [comment]
Since you didn't choose to follow the issue template, I'm assuming the issue you describe has been [fixed](https://github.com/pdoc3/pdoc/commit/0d1e022eeab7aea458151f79367edb5ca6421437) in the most recent release, 0.7.2.

Also maybe consider querying search before opening new issues. :grin:

Duplicate of https://github.com/pdoc3/pdoc/issues/122.
